parsigs/parse_sig_api.py

```diff
@@ -6,6 +6,8 @@
 import re
 import logging
 from spacy import Language
+import translators as ts
+
```

Comment:
@micies great work with Hypothesis! 
remove the old changes and rebase master as there were changes